joomla/joomla-cms
# [5.2] [Guided Tours] Fix permission errors when recording user tour state

Pull Request for Issue #44080.

### Summary of Changes

The Guided Tours component, until now, did not have a dispatcher.
We do not want to have restrictions on the Ajax call when a tour is running so that any user having access to the administrator console can have its tour state recorder.

### Testing Instructions

Create a user with Manager access permissions.
Log as that user.
Either the welcome or the what's new tour should start.

### Actual result BEFORE applying this Pull Request

When dismissing the tour (in any way), you get the error:
![image](https://github.com/user-attachments/assets/d14f6cb2-6c49-4dfe-8c4f-fbb48f379a89)
The state of the tour is not recorded.

### Expected result AFTER applying this Pull Request

No more errors.
On an auto-starting tour, the state of the tour is recorded in #__user_profiles table.
If tours are logged, there should be entries in the action logs.

### Link to documentations
Please select:
